sim/battle.ts

```diff
@@ -726,7 +726,7 @@ export class Battle {
 				// it's changed; call it off
 				continue;
 			}
-			if (effect.effectType === 'Ability' && effect.isBreakable !== false &&
+			if (effect.effectType === 'Ability' && effect.isBreakable &&
```

Comment:
```suggestion
			if (effect.effectType === 'Ability' && effect.isBreakable !== false &&
```
This can't be removed until mods have their Abilities tagged properly.